src/BatteryStateOfChargeService.h

```diff
@@ -44,6 +44,14 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
 private:
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
-
+    double oldcurrent;
+    double newcurrent;
+    double totalcurrent;
+    QTime newtime;
+    QTime oldtime;
+    QTime et;
```

Comment:
`et` could use a better name, without context it's hard to tell what it's supposed to represent! Long names beat cryptic short names everyday :)